src/Versions/V2_1_1/Client/Credentials.php

```diff
@@ -4,26 +4,81 @@
 namespace Chargemap\OCPI\Versions\V2_1_1\Client;
 
 use Chargemap\OCPI\Common\Client\Modules\AbstractFeatures;
-use Chargemap\OCPI\Versions\V2_1_1\Client\Credentials\Register\RegisterCredentialsRequest;
-use Chargemap\OCPI\Versions\V2_1_1\Client\Credentials\Register\RegisterCredentialsResponse;
-use Chargemap\OCPI\Versions\V2_1_1\Client\Credentials\Register\RegisterCredentialsService;
+use Chargemap\OCPI\Versions\V2_1_1\Client\Credentials\Delete\DeleteCredentialsRequest;
+use Chargemap\OCPI\Versions\V2_1_1\Client\Credentials\Delete\DeleteCredentialsResponse;
+use Chargemap\OCPI\Versions\V2_1_1\Client\Credentials\Delete\DeleteCredentialsService;
+use Chargemap\OCPI\Versions\V2_1_1\Client\Credentials\Get\GetCredentialsRequest;
+use Chargemap\OCPI\Versions\V2_1_1\Client\Credentials\Get\GetCredentialsResponse;
+use Chargemap\OCPI\Versions\V2_1_1\Client\Credentials\Get\GetCredentialsService;
+use Chargemap\OCPI\Versions\V2_1_1\Client\Credentials\Post\PostCredentialsRequest;
+use Chargemap\OCPI\Versions\V2_1_1\Client\Credentials\Post\PostCredentialsResponse;
+use Chargemap\OCPI\Versions\V2_1_1\Client\Credentials\Post\PostCredentialsService;
+use Chargemap\OCPI\Versions\V2_1_1\Client\Credentials\Put\PutCredentialsRequest;
+use Chargemap\OCPI\Versions\V2_1_1\Client\Credentials\Put\PutCredentialsResponse;
+use Chargemap\OCPI\Versions\V2_1_1\Client\Credentials\Put\PutCredentialsService;
 
 class Credentials extends AbstractFeatures
 {
-    private ?RegisterCredentialsService $registerCredentialsService = null;
+    private ?PostCredentialsService $postCredentialsService = null;
+    private ?GetCredentialsService $getCredentialsService = null;
+    private ?PutCredentialsService $putCredentialsService = null;
+    private ?DeleteCredentialsService $deleteCredentialsService = null;
 
     /**
-     * @throws \Chargemap\OCPI\Common\Client\Modules\Credentials\Register\ClientAlreadyRegisteredException
+     * @throws \Chargemap\OCPI\Common\Client\Modules\Credentials\ClientAlreadyRegisteredException
+     * @throws \Chargemap\OCPI\Common\Client\OcpiEndpointNotFoundException
+     * @throws \Chargemap\OCPI\Common\Server\Errors\OcpiInvalidPayloadClientError
      * @throws \Psr\Http\Client\ClientExceptionInterface
+     */
+    public function register(PostCredentialsRequest $request): PostCredentialsResponse
```

Comment:
This is a breaking change to some (that the `register` method now requires a `PostCredentialsRequest` instead of the `RegisterCredentialsRequest`).
Since we are doing a breaking change I suggest to rename the methods to to match the other clients as well.
So this could be named `post`

Reply:
Good idea, have fixed it now